### ISSUE TITLE
Electron app

### ISSUE THREAD

**Issue by David-Gil** [mivion/swisseph#54]
Hi!

I'm thinking about learning to program with Electron to make a desktop astrological app. This is the project I'd need to use to make the calculations inside Electron, isn't it?

Thank you for your efforts!!

David

**Comment by timotejroiko** [comment]
yes. i use this project in a node.js application, and i have tested it in a nw.js desktop app as well.
but you have to follow a specific procedure for nwjs and electron. check this:
https://electronjs.org/docs/tutorial/using-native-node-modules
